engo_glfw.go

```diff
@@ -195,7 +197,7 @@ func SetBackground(c color.Color) {
 	if !headless {
 		r, g, b, a := c.RGBA()
 
-		Gl.ClearColor(float32(r), float32(g), float32(b), float32(a))
+		Gl.ClearColor(float32(r)/0xffff, float32(g)/0xffff, float32(b)/0xffff, float32(a)/0xffff)
```

Comment:
This fixes the `engo.SetBackground` on the desktop. It currently only works with 100% black or 100% white. This allows for any color.